sirepo/server.py

```diff
@@ -107,10 +107,13 @@ async def api_deleteSimulation(self):
         simulation_db.delete_simulation(req.type, req.id, qcall=self)
         return self.reply_ok()
 
-    @sirepo.quest.Spec(
-        "require_user", sid="SimId optional", filename="SimFileName", sim_data="SimData"
-    )
+    @sirepo.quest.Spec("require_user", filename="SimFileName")
     async def api_downloadFile(self, simulation_type, simulation_id, filename):
+        """DEPRECATED: Use downloadLibFile"""
+        return await self.api_downloadLibFile(simulation_type, filename)
+
+    @sirepo.quest.Spec("require_user", filename="SimFileName")
+    async def api_downloadLibFile(self, simulation_type, filename):
         # TODO(pjm): simulation_id is an unused argument
```

Comment:
This comment is no longer valid.

Reply:
[c5e937e](https://github.com/radiasoft/sirepo/pull/6348/commits/c5e937e08248ad8a87d685572ecac043b4bbfb9d)